app/controllers/v1/courses_controller.rb

```diff
@@ -81,5 +110,9 @@ def wrong_course_for_teacher
       render json: { error: 'El docente no se relaciona con el curso seleccionado' },
              status: :unprocessable_entity
     end
+
+    def course_params
+      params.require(:course).permit(:name, :vacancies)
```

Comment:
I think you also need the subject_id... You can check this in the schema file where we have all the null restrictions.